Dockerfile

```diff
@@ -0,0 +1,28 @@
+## Go builder image
+FROM golang:1.20.3 as builder
```

Comment:
In my humble opinion, this should match the value provided at:

https://github.com/patrickdappollonio/kubectl-slice/blob/main/go.mod#L3

One way we can do this is potentially passing the argument as part of the process. [Jeff explains it pretty well here](https://www.jeffgeerling.com/blog/2017/use-arg-dockerfile-dynamic-image-specification).

Reply:
This will require more changes than I have time for now.
So skipping this, and renovate will need to handle that part.